src/components/Sidebar/sidebar.scss

```diff
@@ -117,11 +117,11 @@
       opacity: 100%;
       width: 20px;
       height: 20px;
-      margin-left: 12qpx;
+      margin-left: 8px;
     }
   }
   .external-site-link-icon {
-    opacity: 0%;
+    opacity: 0;
```

Comment:
@alexleventer can you debug why this code seems to work on localhost but not in the build environments? It's not an issue introduced by you, but it's something I'm just now noticing.